src/notation/inotationwritersregister.h

```diff
@@ -33,6 +33,8 @@ class INotationWritersRegister : MODULE_EXPORT_INTERFACE
 
     virtual void reg(const std::vector<std::string>& suffixes, INotationWriterPtr writer) = 0;
     virtual INotationWriterPtr writer(const std::string& suffix) const = 0;
+
+    virtual std::vector<std::string> getRegisteredSuffixes() const = 0;
```

Comment:
Please, remove get prefix from the name of this method:

virtual std::vector<std::string> registeredSuffixes() const = 0;